src/sci/impl/namespaces.cljc

```diff
@@ -866,6 +866,7 @@
    're-pattern (copy-core-var re-pattern)
    #?@(:clj ['re-matcher (copy-core-var re-matcher)])
    're-matches (copy-core-var re-matches)
+   #?@(:clj ['realized? (copy-core-var realized?)])
```

Comment:
Why only `:clj`?

Reply:
I didn't know it was part of cljs as well 😅

Reply:
It does look like they are slightly different:

- clj: Returns true if a value has been produced for a promise, delay, future or lazy sequence.
- cljs: Returns true if a value has been produced for a lazy sequence.

but that shouldn't matter 👌